j.podman.md

```diff
@@ -136,6 +136,31 @@ ef4b14a72d02ae0577eb0632d084c057777725c279e12ccf5b0c6e4ff5fd598b
 </p>
 </details>
 
+### Creates a container without running/starting it
+<details><summary>show</summary>
+<p>
+
+```bash
+:~$ podman create simpleapp # create
+:~$ podman container ls
+```
+
+</p>
+</details>
+
+
+### Exports a container to output.tar file
```

Comment:
Can we add the fact that podman is used in the question?

Reply:
I have added the output of the command, would this be sufficient?